packages/modules/viewer-react/src/services/iModel/ViewCreator3d.ts

```diff
@@ -72,6 +73,12 @@ export class ViewCreator3d extends ViewCreator {
               const start = new Date();
               const intvl = setInterval(() => {
                 if (viewPort.areAllTileTreesLoaded) {
+                  ViewerPerformance.addMark("TilesLoaded");
```

Comment:
how helpful will metrics like this be, if we're not tracking with it the number of tiles?

Reply:
the metrics in general are debatable due to so many potential factors. The ask here was to attempt to capture the time for the viewport to be "usable". I'm not sure of a better place for that but certainly open to suggestions.

Reply:
This spot is fine imo, i just think we should also log the number of tiles with it, otherwise, it really is meaningless, and why bother. You should be able to use vp.numReadyTiles

Reply:
https://github.com/iTwin/itwinjs-core/blob/master/test-apps/display-test-app/src/frontend/TileLoadIndicator.ts

the `onFinishRender` looks promising

Reply:
I saw that but its fired in the render loop, which is oddly prior to tile requests: https://github.com/iTwin/itwinjs-core/blob/master/core/frontend/src/IModelApp.ts#L495

Reply:
Nice catch... that is weird. 

So the current event looks fine, thoughts on adding metrics like numReadyTiles?